src/lib/storage/fixedstring_dictionary_column/fixedstring_column_iterable.hpp

```diff
@@ -0,0 +1,117 @@
+#pragma once
```

Comment:
The changes to the regular dictionary iterator look minimal - can't we reuse that one?